src/helpers/openingHoursHelper.ts

```diff
@@ -236,7 +236,7 @@ export const isOpen = (
       .filter((info) => info.weekday === getReadableDay(now) || !info.weekday)
       .filter((info) => isOpeningTimeForDate(info, now));
 
-    const todaysOpenTimes = todaysTimes.filter((info) => !!info.open);
+    const todaysOpenTimes = todaysTimes.filter((info) => !!info.open && !!info.weekday);
```

Comment:
`info.weekday` is this field, or am i wrong?
<img width="746" alt="image" src="https://user-images.githubusercontent.com/1942953/224676187-e92bb345-2c3d-40a9-9320-dc1df40c8408.png">

Reply:
Yes, you are right. Unfortunately, it's not a method we can always trust. But we should definitely check the `weekday`.

Reply:
reworked with 55e3e18, please have a look